src/ViewModels/ArcGisMapServerItemViewModel.js

```diff
@@ -66,6 +66,8 @@ ArcGisMapServerItemViewModel.prototype._enableInCesium = function() {
 
     var scene = this.application.cesium.scene;
 
+    this.legendUrl = this.url + '/legend';
```

Comment:
Well I'm reviewing the code, so I was just suggesting changes to the code (to make it consistent with the WMS item, specifically). :)  No argument that we should get out of the legendUrl business, just like we should get out of the init file business entirely.